code/src/Components/Overview.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+export const Overview= (props) => {
+    const {step, nameInput, emailInput, positionInput, officeInput, skillsArray, linkInput, experienceTitle, onStepBackChange} = props
+    const progress = (step/8)*100
+    return(
+        <div className="container">
+        <div className="overview-container">
+            <h2>SUMMARY</h2>
+            <h3><span className="summaryTitle">Name:</span> {nameInput}</h3>
+            <h3><span className="summaryTitle">Email:</span> {emailInput}</h3>
+            <h3><span className="summaryTitle">Position:</span> {positionInput}</h3>
+            <h3><span className="summaryTitle">Location:</span> {officeInput}</h3>
+            <h3><span className="summaryTitle">Skills:</span> {skillsArray.join(', ')}</h3>
+            <h3><span className="summaryTitle">Experience level:</span> {experienceTitle}</h3>
+            <h3><span className="summaryTitle">Link to portfolio:</span> {linkInput}</h3>
+        </div>
+        <div className="buttons">
+                <input 
+                    className="nextBtn button"
+                    type="submit"
+                    />
```

Comment:
I don't know how picky they are with this but it says in the blue level requirements: 
_"There should be a submit button. When pressed your app should hide the input form and show a summary of the user's submissions."_ 
but you have the submit in the summary which suits your survey better sins that would mean you send the application after you have looked at your answers first. but it's just an observation